I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 17, in script
    with fade # затемнение при переходе
  File "game/script.rpy", line 17, in script
    with fade # затемнение при переходе
OSError: Couldn't find file 'fonk_back.mp3'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "D:\renpy-8.0.3-sdk\renpy\bootstrap.py", line 277, in bootstrap
    renpy.main.main()
  File "D:\renpy-8.0.3-sdk\renpy\main.py", line 660, in main
    run(restart)
  File "D:\renpy-8.0.3-sdk\renpy\main.py", line 146, in run
    renpy.execution.run_context(True)
  File "D:\renpy-8.0.3-sdk\renpy\execution.py", line 938, in run_context
    context.run()
  File "game/script.rpy", line 17, in script
    with fade # затемнение при переходе
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python3.9/site-packages/future/utils/__init__.py", line 441, in raise_
  File "game/script.rpy", line 17, in script
    with fade # затемнение при переходе
  File "D:\renpy-8.0.3-sdk\renpy\ast.py", line 1658, in execute
    renpy.exports.with_statement(trans, paired=paired)
  File "D:\renpy-8.0.3-sdk\renpy\exports.py", line 1657, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "D:\renpy-8.0.3-sdk\renpy\display\core.py", line 2798, in do_with
    return self.interact(trans_pause=True,
  File "D:\renpy-8.0.3-sdk\renpy\display\core.py", line 3377, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "D:\renpy-8.0.3-sdk\renpy\display\core.py", line 4103, in interact_core
    renpy.audio.audio.periodic()
  File "D:\renpy-8.0.3-sdk\renpy\audio\audio.py", line 1166, in periodic
    raise_(exc[0], exc[1], exc[2])
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python3.9/site-packages/future/utils/__init__.py", line 441, in raise_
  File "D:\renpy-8.0.3-sdk\renpy\audio\audio.py", line 1144, in periodic_thread_main
    periodic_pass()
  File "D:\renpy-8.0.3-sdk\renpy\audio\audio.py", line 1086, in periodic_pass
    c.periodic()
  File "D:\renpy-8.0.3-sdk\renpy\audio\audio.py", line 517, in periodic
    topf = load(filename)
  File "D:\renpy-8.0.3-sdk\renpy\audio\audio.py", line 70, in load
    rv = renpy.loader.load(fn)
  File "D:\renpy-8.0.3-sdk\renpy\loader.py", line 820, in load
    raise IOError("Couldn't find file '%s'." % name)
OSError: Couldn't find file 'fonk_back.mp3'.

Windows-10-10.0.19045 AMD64
Ren'Py 8.0.3.22090809
nicetest 1.0
Wed Nov 23 15:08:01 2022
